### PR TITLE
Add hacktoberfest program. Remove year from program titles. Delete gs…

### LINKED ISSUE #829: Program card for Hacktoberfest
**Description**

Create a program card for Hacktoberfest

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by jesuslerma**
**Description**
I added a new program for hacktoberfest 2020 participation.
This PR fixes #829 

Hi @prateeksahu and @chandrashritii can u please review this entry?
**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by jesuslerma** [comment]
Hi I'm not shure why in the preview is redirecting all the links to production layer5.io site. But you can look at the entry here: https://5f72907fc763220009305fc0--layer5io.netlify.app/programs/hacktoberfest

File: collections/_programs/hacktoberfest_2020.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    </ol>
+    <ol>
+      We're preparing an interesting list of issues that will be good for this event. You can found that list <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> here </a>
+    </ol>
+    <ol>
+      <b> Fill the Questionnaire about Open Source </b>
+    </ol>
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+      </p>
```
Comment (Jashpatel1) [inline collections/_programs/hacktoberfest_2020.md:46]: This closing tag is extra. Can you please remove it @jesuslerma

File: collections/_programs/hacktoberfest_2020.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      projects to provide learning environments, deployment and operational best
+      practices, performance benchmarks, create documentation, share networking
+      opportunities, and more. Our shared commitment to the open source spirit
+      pushes Layer5 projects forward. New members are always welcome.
+    </ol>
+    <ol>
+      <b> Layer5 has a couple of issues ready for hacktoberfest's participants </b>
+    </ol>
+    <ol>
+      We're preparing an interesting list of issues that will be good for this event. You can found that list <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> here </a>
```
Comment (Jashpatel1) [inline collections/_programs/hacktoberfest_2020.md:39]: @jesuslerma  Can you please update the line as "You can find the list ..." ?

File: collections/_programs/hacktoberfest_2020.md (hunk 1 above)
Comment (jesuslerma) [reply collections/_programs/hacktoberfest_2020.md]: Done :)

File: collections/_programs/hacktoberfest_2020.md (hunk 2 above)
Comment (jesuslerma) [reply collections/_programs/hacktoberfest_2020.md]: Done :)

**Review by Jashpatel1 — APPROVED** [review]

**Comment by Jashpatel1** [comment]
Nice work here @jesuslerma. Thanks for taking it up with priority.

**Comment by jesuslerma** [comment]
You're welcome @Jashpatel1 I'm very happy to help :)

**Review by tripathyprateek — DISMISSED** [review]
Looks good to me. All required changes have been done. Great job @jesuslerma

File: collections/_programs/hacktoberfest_2020.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    <ol>
+      We're preparing an interesting list of issues that will be good for this event. You can find the list <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> here </a>
+    </ol>
+    <ol>
+      <b> Fill the Questionnaire about Open Source </b>
+    </ol>
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+
+      Please fill <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">this questionnaire </a> so our community can be prepared for a harm welcome for you and other new members.
```
Comment (chandrashritii) [inline collections/_programs/hacktoberfest_2020.md:47]:
```suggestion
      Help us better prepare to support you as you find your way into the world of Open Source by filling up this <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">questionnaire</a>.
```

File: collections/_programs/hacktoberfest_2020.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    </ol>
+  </ul>
+Pull requests can be made in any GitHub-hosted repositories/projects.
```
Comment (chandrashritii) [inline collections/_programs/hacktoberfest_2020.md:50]:
```suggestion
Pull requests can be made in any of the Layer5 GitHub-hosted repositories/projects.
```

File: collections/_programs/hacktoberfest_2020.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+You can sign up anytime between October 1 and October 31. You can start hacking <a href="https://hacktoberfest.digitalocean.com">here</a>.
```
Comment (chandrashritii) [inline collections/_programs/hacktoberfest_2020.md:51]:
```suggestion
You may sign up anytime between 1st to 31st October. Begin hacking <a href="https://hacktoberfest.digitalocean.com">here</a>!
```

**Review by chandrashritii — CHANGES_REQUESTED** [review]
Great work here. Thank you @jesuslerma A few editorial changes :)

**Comment by jesuslerma** [comment]
Hi @chandrashritii all your requested changes have been solved. Please take a look, thanks!

**Review by chandrashritii — APPROVED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
Screen Shot 2020-09-29 at 10 51 41 AM

Hacktoberfest logo should be converted to grayscale to match the rest of the logos.

**Comment by leecalcote** [comment]
Let's do what we can to avoid hyperlinks to "here". Instead hyperlink the thing itself.

**Comment by leecalcote** [comment]
@chandrashritii where is the GSoC 2019 information on or linked to from this page? https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

**Comment by Nikhil-Ladha** [comment]
I think something has been broken here, the link to any of the programs is not working anymore. Can you just check it once?
Also, shouldn't we make the hacktoberfest icon `grayscale` too? Like the other programs.

**Comment by jesuslerma** [comment]
Hi @Nikhil-Ladha  I think there is an error in the configuration of those program cards because each link on card is redirecting to https://layer5.io/programs/* for example if you clic on hactktoberfest card it will redirect to https://layer5.io/programs/hacktoberfest. The rest of the programs was renamed to gsod/ instead of gsod/2020 for example.

**Comment by jesuslerma** [comment]
> @chandrashritii where is the GSoC 2019 information on or linked to from this page? https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

Hi I lee I removed the GSoC 2019 card program from /programs main page because when I added the new card the layout doesn't fit well. So I asked Shriti and she approved to me to delete that program. Or maybe I didn't understand and the entry shouldn't be removed. 

Please let me know if it's ok to remove the 2019 program entry, or maybe I should add the existingt program into this entry: https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

**Comment by Nikhil-Ladha** [comment]
> Hi @Nikhil-Ladha I think there is an error in the configuration of those program cards because each link on card is redirecting to https://layer5.io/programs/* for example if you clic on hactktoberfest card it will redirect to https://layer5.io/programs/hacktoberfest. The rest of the programs was renamed to gsod/ instead of gsod/2020 for example.

Ohh, gotcha. No, the issue is with netlify deployment it happened previously too.
Can you just confirm if the pages work fine in the local deployment?

**Comment by chandrashritii** [comment]
> 
> 
> > @chandrashritii where is the GSoC 2019 information on or linked to from this page? https://deploy-preview-852--layer5io.netlify.app/programs/gsoc
> 
> Hi I lee I removed the GSoC 2019 card program from /programs main page because when I added the new card the layout doesn't fit well. So I asked Shriti and she approved to me to delete that program. Or maybe I didn't understand and the entry shouldn't be removed.
> 
> Please let me know if it's ok to remove the 2019 program entry, or maybe I should add the existingt program into this entry: https://deploy-preview-852--layer5io.netlify.app/programs/gsoc

Yes @jesuslerma  As discussed, please add a link to the GSoC 2019 page to the GSoC (previously titled GSoC 2020) page.

**Comment by jesuslerma** [comment]
Hi @chandrashritii and @leecalcote  I just added the prev gsoc for 2019. Also I updated the HF logo to be in grayscale, and I updated all my hyperlinks to be redirected to their real values. Please take a look.

**Comment by jesuslerma** [comment]
@Nikhil-Ladha yes all the changes are working in my localmachine :+1:

**Comment by chandrashritii** [comment]
> 
> 
> Hi @chandrashritii and @leecalcote I just added the prev gsoc for 2019. Also I updated the HF logo to be in grayscale, and I 
updated all my hyperlinks to be redirected to their real values. Please take a look.

Hey @jesuslerma Did you commit these changes? I don't see any new commits

**Comment by jesuslerma** [comment]
hi @chandrashritii  sorry I forgot to push the changes. Please take a look again

File: collections/_programs/hacktoberfest_2020.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+    <ol>
+      Find a list of beginner friendly issues here: <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues </a> to make your mark in open-source and move towards successfully completing the Hacktoberfest challenge.
+    </ol>
+    <ol>
+      <b> Fill the Questionnaire about Open Source </b>
+    </ol>
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+
+      Help us better prepare to support you as you find your way into the world of Open Source by filling up this questionnaire: <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform</a>
```
Comment (tripathyprateek) [inline collections/_programs/hacktoberfest_2020.md:47]: Hi @jesuslerma could you just add this Hacktoberfest Meetup event link to this page as well 
`https://organize.mlh.io/participants/events/4796-getting-started-layer5`

**Comment by chandrashritii** [comment]
LGTM

**Review by leecalcote — APPROVED** [review]
Thanks, @jesuslerma! 👍

**Comment by leecalcote** [comment]
Haha! 🎈

**Pull request merged**